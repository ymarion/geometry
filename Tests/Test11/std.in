# Initializing 2 objects
C c1 0 0 10
C c2 10 10 10
# Undoing last operation
UNDO
# Should just show c1
LIST


