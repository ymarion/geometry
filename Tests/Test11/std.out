# Creations
OK
OK
# Undoing
OK
C c1 0 0 10
